Update application-gateway-configure-redirect-powershell.md

ShuheiUda opened the pull request:
Remove "-IncludePath $true -IncludeQueryString $true"

> The following example redirects traffic coming into the application gateway to an external website (bing.com). When using the `TargetUrl` switch the `IncludePath` and `IncludeQuery` switches are not allowed.

davidwrede commented:
Thanks for the contribution, Josh. Because you're a Microsoft employee, please submit any future changes through our private repo.

Thanks,
Dave

ShuheiUda committed:
Update application-gateway-configure-redirect-powershell.md

Remove "-IncludePath $true -IncludeQueryString $true"

> The following example redirects traffic coming into the application gateway to an external website (bing.com). When using the `TargetUrl` switch the `IncludePath` and `IncludeQuery` switches are not allowed.
